flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -734,6 +789,19 @@ public class TaskManagerOptions {
                                                     code(TaskManagerLoadBalanceMode.NONE.name())))
                                     .build());
 
+    /** Type of redirection of {@link System#out} and {@link System#err}. */
+    public enum SystemOutMode {
+
+        /** Don't change the System.out and System.err, it's the default value. */
+        DEFAULT,
+
+        /** Redirect System.out and err to LOG. */
+        LOG,
+
+        /** Ignore all System.out and err directly. */
+        IGNORE
```

Comment:
could we maintain a consistent style in '{@link System#out/err' here?

Reply:
Thanks for your review!

Good catch, and updated.